core/src/main/java/com/digitalpebble/storm/crawler/spout/RandomURLSpout.java

```diff
@@ -42,15 +43,14 @@
     private boolean active = true;
     private boolean removeAfterSending = true;
 
-    private String[] urls = new String[] { "http://www.lequipe.fr/",
-            "http://www.lemonde.fr/", "http://www.bbc.co.uk/",
-            "http://www.facebook.com/", "http://www.rmc.fr" };
-
-    public RandomURLSpout(String... urls) {
```

Comment:
This constructor is very useful. It should be kept and populate the HashMap just like the default constructor does.